app/views/notifications/_notifications.html.erb

```diff
@@ -1,5 +1,9 @@
 <div id="unapproved-sign-slide-<%= notification.id %>" class="unapproved-sign-slide">
-  <p><%= image_tag notification.slide.screenshot, class: 'screenshot' %></p>
+  <p>
+    <div class="slide-preview-wrapper">
```

Comment:
Now that I look at this, I think you can test this. Just test that `div.slide-preview-wrapper` is present with iframe when notification should have preview.